Mathlib/AlgebraicGeometry/Limits.lean

```diff
@@ -47,6 +47,9 @@ instance : IsAffine (⊤_ Scheme.{u}) :=
 instance : HasFiniteLimits Scheme :=
   hasFiniteLimits_of_hasTerminal_and_pullbacks
 
+instance (X : Scheme.{u}) : X.Over (⊤_ _) := ⟨terminal.from _⟩
+instance {X Y : Scheme.{u}} (f : X ⟶ Y) : f.IsOver (⊤_ _) := ⟨terminal.hom_ext _ _⟩
+
```

Comment:
These could also go in the general part, might need additional imports though. Do however you prefer.

Reply:
I don't think we should provide these instances for arbitrary categories.